chef/cookbooks/apache2/recipes/default.rb

```diff
@@ -222,8 +222,3 @@
 
 # We don't need this.
 #apache_site "default" if platform?("centos", "redhat", "fedora")
-
-service "apache2" do
-  action :start
-  ignore_failure true
-end
```

Comment:
That's why I added https://github.com/crowbar/crowbar-core/pull/69#issuecomment-146556700 . The think is that I don't find a nice way to fix it as long as ```a2enmod``` is broken for a graceful apache restart